### ISSUE TITLE
β-ketones support

### ISSUE THREAD

**Issue by leleobhz** [glucometers-tech/glucometerutils#18]
Hello dev!

I have a FreeStyle Optium Neo mainly to measure Ketones. I found you code and using it I found the values for my Ketones.

Using a print in results after https://github.com/Flameeyes/glucometerutils/blob/master/glucometerutils/drivers/fsprecisionneo.py#L68 i've found the following dict:

```
 leonardo  ~  bin  glucometerutils  ./glucometer.py --driver fsprecisionneo dump
['9', '13', '7', '18', '17', '18', '33', '1', '6', '1', '0']
['9', '4', '7', '18', '17', '14', '36', '1', '1', '1', '0']
['6', '2', '7', '18', '17', '14', '32', '1', '7', '18', '17', '14', '5']
[]
```

The result for 18:33 was 0.3 mmol/l and the result for 14:36 was 0.1 mmol/l. My unit is mg/dl for glucose but mmol/l for Ketones. May this help with unknown values?

Anyways, I did the following modification after https://github.com/Flameeyes/glucometerutils/blob/master/glucometerutils/drivers/fsprecisionneo.py#L34 :

```
_TYPE_GLUCOSE_READING = '9'

_NeoReading = collections.namedtuple('_NeoReading', (
    'type',  # 7 = blood glucose
    'id',
    'month', 'day', 'year',  # year is two-digits
    'hour', 'minute',
    'unknown2',
    'value',
    'unknown3', 'unknown4',
))
```

And modified the https://github.com/Flameeyes/glucometerutils/blob/master/glucometerutils/drivers/fsprecisionneo.py#L64 to _return common.UNIT_MMOLL_ - resulting the following dump:

```
 leonardo  ~  bin  glucometerutils  ./glucometer.py --driver fsprecisionneo dump
['9', '13', '7', '18', '17', '18', '33', '1', '6', '1', '0']
['9', '4', '7', '18', '17', '14', '36', '1', '1', '1', '0']
['6', '2', '7', '18', '17', '14', '32', '1', '7', '18', '17', '14', '5']
[]
"2017-07-18 14:36:00","0.06","",""
"2017-07-18 18:33:00","0.33","",""

```

Just for information, at 14:32 I did the first configuration to my glucosimeter.

So, With this, can you please:

* Add support do dump more than one type of recorded (Maybe with _TYPE_GLUCOSE_READING as a tuple?)
* Add suport to this ketones reading?

I can extract more data when I do more measurements. Also, I can implement it too, but can take some time to be done (Maybe ~2 weeks if my job allows me some time).

Thanks!

**Comment by scrool** [comment]
Isn't this duplicate/already implemented with #19?

**Comment by Flameeyes** [comment]
It is indeed.
